src/Catalyst.Cli.Tests/UnitTests/Commands/Response/RemovePeerResponseTests.cs

```diff
@@ -42,7 +42,7 @@ public void RemovePeerResponse_Can_Get_Output()
             //Arrange
             var removePeerResponse = new RemovePeerResponse();
             var commandContext = TestCommandHelpers.GenerateCliResponseCommandContext(_testScheduler);
-            var removePeerCommand = new PeerRemoveCommand(commandContext, Substitute.For<ILogger>());
+            new PeerRemoveCommand(commandContext, Substitute.For<ILogger>());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either remove this useless object instantiation of class 'PeerRemoveCommand' or use it.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)